joomla/joomla-cms
# [com_media] Alignment and spacing fixes

Pull Request for Issue # .

### Summary of Changes
General alignment and spacing fixes to the media manager

### Testing Instructions
Apply patch and navigate to media manager. Ensure that elements are in line with each other. Navigate to a sub folder and ensure the back button is vertically centered within its container.

### Before Patch
![media1](https://cloud.githubusercontent.com/assets/2803503/23830765/430a2f34-070a-11e7-96b9-bbe9949397a4.png)

### After Patch
![media2](https://cloud.githubusercontent.com/assets/2803503/23830767/4e3fe72c-070a-11e7-9670-c38a610bcafb.png)

### Documentation Changes Required
None
